enhancements/vdpa/vdpa-introduction.md

```diff
@@ -140,6 +107,10 @@ Runtime errors (user creates a combination of fields that are wrong) are more di
 
 ### Implementation Details/Notes/Constraints
 
+The proposed implementation depends on the CX-6 OVS hardware work which is not fully ready yet in OCP 4.11.
```

Comment:
Better to be specific. There are CX-6 Dx, CX-6 Lx and CX-6. Theses are 3 different cards.